minecraft_servers/vinixmc/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "server_name": "vinixmc",
+  "nice_name": "VinixMC Network",
+  "direct_ip": "play.vinixmc.net",
+  "server_wildcards": [
+    "%.vinixmc.net"
+  ],
+  "social": {
+    "web_shop": "https://store.vinixmc.net",
+    "twitter": "16a2",
+    "discord": "https://discord.vinixmc.net"
```

Comment:
Custom links are reserved for partners only; use a normal discord.gg/[INVITE] link instead